src/wix/wix/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I don't think we should add launchSettings.json generally. :)

Reply:
It's a temporary addition and will be removed before the final addition.